apps/site/src/lib/components/Navbar/Navbar.module.scss

```diff
@@ -0,0 +1,58 @@
+.navMenuListWrapper {
+	// clip top only when mobile
+	@media (max-width: 768px) {
+		clip-path: inset(0px 0px -100px -100px);
+	}
+}
+
+.navMenuList {
+	@media (max-width: 768px) {
+		transform: translateY(-110%);
+		> * {
+			margin-bottom: 1rem;
+		}
+	}
+}
+
+// so the transition can be disabled on media break
+.transformTransition {
+	transition: transform 0.5s ease-in-out;
+}
+
+.hideList {
+	@media (max-width: 768px) {
+		transform: translateY(-110%);
+	}
+}
+
+.showList {
+	@media (max-width: 768px) {
+		transform: translateY(0px);
+	}
+}
```

Comment:
Nitpick: For styles as simple as these, please see if it's possible to use Tailwind classes instead. See the Tailwind docs for [translating](https://tailwindcss.com/docs/translate), for example.